configs/openmrs_config/encountertypes/encountertypes_core.csv

```diff
@@ -12,6 +12,6 @@ d7151f82-c1f3-4152-a605-2f9ea7414a79,,Visit Note,,,"Encounter where a full or ab
 3596fafb-6f6f-4396-8c87-6e63a0f1bd71,,Lab Results,,,Encounter for capturing lab results,,,,
 39da3525-afe4-45ff-8977-c53b7b359158,,Order,,,Encounter for capturing orders,,,,
 b870b6bb-9441-4664-9087-3d3477174213,,Adult NCD Consultation,Adult NCD Consultation,ការពិគ្រោះយោបល់ជំងឺមិនឆ្លង សម្រាប់មនុស្សពេញវ័យ,,,,,
-f9fdbe63-12b4-479f-b8fd-dcbfde6aa944,,Adult NCD Registration,Adult NCD Registration,ទម្រង់ពិនិត្យស្រាវជ្រាវរុករកជំងឺមិនឆ្លង សម្រាប់មនុស្សពេញវ័យ,,,,,
+f9fdbe63-12b4-479f-b8fd-dcbfde6aa944,,NCD Medical History,NCD Medical History,ប្រវត្តជំងឺ,,,,,
```

Comment:
Could there be a link that redirects to the sheet that contains the equivalent Khmer translation?

Reply:
This has been provided

Reply:
Noted @rbuisson , thank you for this